02-Appetisers/Barotropic_Streamfunction.ipynb

```diff
@@ -2,12 +2,20 @@
  "cells": [
```

Comment:
<p>The <code># python</code> lines could be removed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/617/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>